models/endoscopy_inbody_classification/docs/readme.md

```diff
@@ -0,0 +1,129 @@
+# Description
+A pre-trained model for the endoscopy inbody classification task.
+
+# Model Overview
+This model is trained using the SEResNet50 structure, whose details can be found in [1]. All datasets are from private samples of [Activ Surgical](https://www.activsurgical.com/). Samples in training and validation dataset are from the same 4 videos, while test samples are from different two videos.
+The [pytorch model](https://drive.google.com/file/d/14CS-s1uv2q6WedYQGeFbZeEWIkoyNa-x/view?usp=sharing) and [torchscript model](https://drive.google.com/file/d/1fOoJ4n5DWKHrt9QXTZ2sXwr9C-YvVGCM/view?usp=sharing) are shared in google drive.
+
+## Data
+Datasets used in this work were provided by [Activ Surgical](https://www.activsurgical.com/). Here is a [link](https://drive.google.com/uc?id=1rQQfHcZFs74OG0uJsL4vg7YqMWXV-s4k) of 20 samples (10 in-body and 10 out-body) to show what this dataset looks like.
+
+The input label json should be a list made up by dicts which includes "image" and "label" keys. An example format is shown below.
+
+```
+[
+    {
+        "image":"/path/to/image/image_name0.jpg",
+        "label": 0
+    }
```

Comment:
There needs to be a , after each }

Reply:
Hi @jin-nvidia :
    Thanks for your advice! I have added commas in this part.
Thanks
BIn